testsuite/MDAnalysisTests/analysis/test_helanal.py

```diff
@@ -198,14 +198,13 @@ def test_warnings(tmpdir):
                 u, select="name CA", begin=-1, finish=99999
             )
 
-            assert len(record) == 2
+            assert len(record) == 3
```

Comment:
I guess that works as a really minimal test 👍

Reply:
Curious that DeprecationWarning counts as a UserWarning... but ok, if it works...